index.html

```diff
@@ -2128,6 +2291,220 @@ <h4>Bidirectional text</h4>
 
     </section> <!-- Profile Constraints -->
 
+    <section id="ttml-to-data-model">
+      <h2>Mapping from TTML to the DAPT Data Model</h2>
+      <p><a href="#data-model"></a> defines how objects and properties of the DAPT data model are represented in TTML, i.e. in a <a>DAPT Document</a>.
+        However, the set of possible representations is not fully specified.</p>
```

Comment:
I would suggest rephrasing this sentence. It gives the impression that the spec is not complete by saying "not fully specified". I would suggest being more explicit such as:
```suggestion
         However, a given DAPT data model can possibly lead by multiple TTML representations."
```

Reply:
Reworded in 1e6bd9b.